pkgs/development/python-modules/meep/default.nix

```
@@ -35,13 +35,13 @@ assert !lapack.isILP64;
 
 buildPythonPackage rec {
   pname = "meep";
-  version = "1.30.0";
+  version = "1.30.1";
 
   src = fetchFromGitHub {
     owner = "NanoComp";
     repo = pname;
```

```suggestion
    repo = "meep";
```
https://github.com/NixOS/nixpkgs/issues/277994

---

Thanks for the comment, but I suggest fixing these issues in a separate PR.